app/components/domain-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, HStack, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+
+export default function DomainForm() {
+  return (
+    <Form className="domain-form" method="post">
+      <VStack maxW="xl" spacing="5">
+        <HStack w="100%">
+          <Input placeholder="Domain Name" name="name" />
```

Comment:
Every form element needs a label.  You can't use the `placeholder` as a label because as soon as the user starts typing, they lose the info about what the field represents.  You can use the `placeholder` to give an example of valid input.

Reply:
Hmmm, the design does not have labels but only uses placeholders. Should I add labels at the top of the fields? Any thoughts @Myrfion @Ririio ?

Reply:
The design is wrong.  Decades of work on usability of forms says that you need to have a label for each form element.  There are a few reasons:

1. on mobile, the label is `for="id-of-input"` which increases the touch area (i.e., touch the label or input and get focus)
2. placeholder-labels disappear as soon as there is text entered.  If a user has multiple fields like this, and an error "incorrect input" they have no idea which field is the wrong one.  You need to make the form navigable after it has data included

Reply:
Yeah, I will add the labels at the top of the fields for now

Reply:
Agree with @humphd  on that, we need labels to be labels, not placeholders